src/test/README.md

```diff
@@ -53,6 +53,11 @@ The `-printtoconsole=1` after the two dashes redirects the debug log, which
 would normally go to a file in the test datadir
 (`BasicTestingSetup::m_path_root`), to the standard terminal output.
 
+The `TEST_BITCOIN_PATH` environment variable, if set, determines the
```

Comment:
I think it's helpful to have all env vars start with `BITCOIN_`
```suggestion
The `BITCOIN_TEST_PATH` environment variable, if set, determines the
```

Reply:
Good idea, I named it that way because the executable is called `test_bitcoin` but that's not a strong reason. I'll make that change.